app/utils/transactionCosts.ts

```diff
@@ -75,7 +76,10 @@ export function calculateCost(
     );
 }
 
-function getPayloadSizeEstimate(transactionKind: TransactionKindId) {
+/**
+   Note that this function does not support scheduled transfers nor transfers with memos.
```

Comment:
It would be good to document what the function does now that you have written what it _does  not_.